src/main/java/com/bunq/sdk/model/generated/PaymentChat.java

```diff
@@ -108,14 +108,20 @@
     return list(apiContext, userId, monetaryAccountId, paymentId, new HashMap<>());
   }
 
+  public static BunqResponse<List<PaymentChat>> list(ApiContext apiContext, Integer userId,
+      Integer monetaryAccountId, Integer paymentId, Map<String, String> params) {
+    return list(apiContext, userId, monetaryAccountId, paymentId, params, new HashMap<>());
+  }
+
   /**
    * Get the chat for a specific payment.
```

Comment:
And now `payment` with lower case `p`  🤔 I see some inconsistency :grin:

Reply:
generated